tests/unit-test.el

```diff
@@ -141,31 +141,132 @@
 
 (describe "cljr--unresolved-alias-ref"
   (it "returns unresolved alias reference"
-    (expect (cljr--with-clojure-temp-file "foo.clj"
-              (insert "(ns foo)")
-              (cljr--unresolved-alias-ref "ns-name"))
-            :to-equal "ns-name"))
+    (cljr--with-clojure-temp-file "foo.clj"
+      (insert "(ns foo)")
+      (expect (cljr--unresolved-alias-ref "ns-name")
+              :to-equal "ns-name")))
 
   (it "returns nil for resolved alias"
-    (expect (cljr--with-clojure-temp-file "foo.clj"
-              (insert "(ns foo (:require [user.ns-name :as ns-name]))")
-              (cljr--unresolved-alias-ref "ns-name"))
-            :to-be nil))
+    (cljr--with-clojure-temp-file "foo.clj"
+      (insert "(ns foo (:require [user.ns-name :as ns-name]))")
+      (expect (cljr--unresolved-alias-ref "ns-name")
+              :to-be nil)))
 
   (it "returns nil for js alias in cljs file."
-    (expect (cljr--with-clojure-temp-file "foo.cljs"
-              (insert "(ns foo)")
-              (cljr--unresolved-alias-ref "js"))
-            :to-be nil))
+    (cljr--with-clojure-temp-file "foo.cljs"
+      (insert "(ns foo)")
+      (expect (cljr--unresolved-alias-ref "js")
+              :to-be nil)))
 
   (it "returns js for js alias in clj file."
-    (expect (cljr--with-clojure-temp-file "foo.clj"
-              (insert "(ns foo)")
-              (cljr--unresolved-alias-ref "js"))
-            :to-equal "js"))
+    (cljr--with-clojure-temp-file "foo.clj"
+      (insert "(ns foo)")
+      (expect (cljr--unresolved-alias-ref "js")
+              :to-equal "js")))
 
   (it "returns js for js alias in cljc file."
-    (expect (cljr--with-clojure-temp-file "foo.cljc"
-              (insert "(ns foo)")
-              (cljr--unresolved-alias-ref "js"))
-            :to-equal "js")))
+    (cljr--with-clojure-temp-file "foo.cljc"
+      (insert "(ns foo)")
+      (expect (cljr--unresolved-alias-ref "js")
+              :to-equal "js"))))
+
+(defmacro with-point-at (text &rest body)
+  (declare (indent 1))
+  `(progn (insert ,text)
+          (goto-char (point-min))
+          (re-search-forward "|")
```

Comment:
Let's use `<POINT>` as the marker, which although ugly, is unmistakable and will result in pretty self-describing tests IMO

Reply:
I cribbed this approach from `clojure-mode`, `cider`, and `smartparens` tests. I would much prefer to match a common formatting scheme over inventing our own. You are welcome to change how you see fit though.

Reply:
> I cribbed this approach from clojure-mode, cider, and smartparens tests.

Ok, let's go with it.